data/apdb-ap-pipe-schema-extra.yaml

```diff
@@ -616,4 +616,6 @@ columns:
   type: BOOL
   nullable: true
   description: Object determined to be a dipole.
-
+- name: bboxSize
+  type: BIGINT
+  nullable: true
```

Comment:
This comes from afw table schema (https://github.com/lsst/ap_association/blob/master/python/lsst/ap/association/afwUtils.py#L716-L718) that was used before to specify schema (merged with YAML files) and I wanted to keep schema exactly the same when afw.table schema disappeared.

BTW, I do not know if `afwUtils.py` is still needed. I kept it because one of the methods there is used by `test_diaForcedSource.py`, but I have no idea if that test is till useful. If you tell me to drop `afwUtils.py` and remove that unit test I can do it on this ticket as well.

Reply:
👍 for the `bboxSize`.

For `afwUtils.py`, I suggest asking @morriscb.

Reply:
@morriscb says `afwUtils.py` is no longer useful, so I'll just remove it in a separate commit.